modules/nf-core/emu/abundance/meta.yml

```diff
@@ -47,7 +47,7 @@ output:
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
       pattern: "*{.tsv}"
-  - results/*read-assignment-distributions.tsv:
+  - '*read-assignment-distributions.tsv':
```

Comment:
This meta for the TSV itself is still copy-pasta- from meta

Reply:
Updated.